automation/build-bin.ts

```diff
@@ -104,9 +264,13 @@ export async function buildOclifInstaller() {
 			await Bluebird.fromCallback(cb => rimraf(dir, cb));
 		}
 		console.log('=======================================================');
-		console.log(`oclif-dev "${packCmd}" [${packOpts}]`);
+		console.log(`oclif-dev "${packCmd}" "${packOpts.join('" "')}"`);
+		console.log(`cwd="${process.cwd()}" ROOT="${ROOT}"`);
```

Comment:
This was added for debugging but I'd just leave it there. The build output has thousands and thousands of output lines... Having these few extra `console.log` won't hurt and can be quite useful to understand and reproduce some issues in `fly intercept` shell sessions.